src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -210,6 +210,63 @@ class C
             Assert.Null(cmd.AnalyzerOptions);
         }
 
+        [Theory, CombinatorialData, WorkItem("https://github.com/dotnet/roslyn/issues/72657")]
+        public void AnalyzerConfig_DoubleSlash(bool doubleSlash1, bool doubleSlash2)
```

Comment:
nit: consider naming these parameters based on which element we are introducing double slashes for, e.g. `bool doubleSlashAnalyzerConfig, bool doubleSlashSource`